docs/UserGuide.md

```diff
@@ -50,110 +70,181 @@ Some example commands you can try:
 
 ---
 
-### Requesting help URL: `help`
+### View help: `help`
+
+**Description:** Are you new or confused with the commands? View the command summary and user guide through a quick pop up window!
+
+**Format:** `help`
 
-Shows a URL that redirects the user to Trackermon's user guide.
+**Example & Output:** `help`
 
-Format: `help`
+[INSERT IMAGE AFTER UI IS DONE COMPLETELY]
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
 ---
 
 ### Finding a show: `find`
 
-**General Find**: Finds shows in Trackermon by matching the user's input across the name, status and tag parameters.
+**Description (General Find):** Searching for a show across all parameters? Find shows containing the search words.
 
-Format: `find <KEYWORD>`
-* Find shows with the specified `<KEYWORD>`.
-* The keyword refers to the input entered by the user after `find`.
-* The keyword **can be a word or number** such as hero, S1,...
-* There must be at **least one keyword** and it **must not be empty**.
+**Format (General Find):** `find KEYWORD`
 
-Examples:
-* `find shingeki` displays all the shows in the list that contain the keyword `shingeki` whether it is a name, status or tag.[put image here after ui update]
-* `find shingeki no kyojin` displays all the shows in the list that contain the keyword `shingeki` **or** `no` **or** `kyojin` whether it is a name, status or tag.[put image here after ui update]
+**Example & Output:** `find attack on titan`
 
-**Precise Find** Finds shows in Trackermon by matching the user's input across name, status and tag parameters with prefixes.
+[INSERT IMAGE AFTER UI IS DONE COMPLETELY]
 
-Format: `find n/[NAME] s/[STATUS] t/[TAG]`
-* **Within a single prefix**, the find command will execute an **AND** search across Trackermon's show list and return all shows that match all keywords that are input by the user in the specific prefix.
-* **Across multiple prefixes**, the find command will execute an **AND** search across Trackermon's show list and return all shows that match all the keywords across all prefixes.
-* There must be at **least one prefix field** and it **must not be empty**.
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about general find:**<br>
+* The `KEYWORD` refers to the input you enter after `find`.
+* `find` must be followed with a space before entering the `KEYWORD`.
+* The `KEYWORD` **can be a word or number** such as hero, S1,...
+* The `KEYWORD` must contain **at least one word** and it **must not be empty**.
+* `find attack on titan` displays all the shows in the list that contain the keywords `attack`, `on` or `titan`, whether it is a name, status or tag.
+
+</div>
+
+**Description (Precise Find):** Searching for a show across specific parameters? Find shows containing the search words based on [prefix](#command-structure).
+
+**Format (Precise Find):** `find n/[NAME] s/[STATUS] t/[TAG]`
```

Comment:
Shouldnt this be `[n/NAME] [s/Status] [t/TAG]` instead?

Reply:
Good spot.